mlflow/sagemaker/deploy.py

```diff
@@ -57,7 +57,7 @@ def _deploy(role, container_name, app_name, model_s3_path, run_id):
     :param run_id:
     :return:
     """
-    sage_client = boto3.client('sagemaker', region_name="us-west-2")
+    sage_client = boto3.client('sagemaker', region_name)
```

Comment:
Are you sure the second argument is `region`? [This documentation](https://boto3.readthedocs.io/en/latest/reference/core/boto3.html) would seem to indicate that everything but the name is a **kwargs, so it would need to be named -- i.e., `region_name= region_name`.

Reply:
Ah -- I see it now, it is actually the second argument from [here](https://boto3.readthedocs.io/en/latest/reference/core/session.html#boto3.session.Session.client). Since it has a default value of None, would you mind adding the explicit `region_name= region_name` anyway, just to be 100% clear?